CRITICAL - 2022-11-28 11:17:48 --> Invalid file: v_pendaftaran/nden.php
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#2 D:\Xampp\htdocs\app-pmb2\app\Controllers\Pendaftaran.php(550): view()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pendaftaran->nde()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#6 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-11-28 11:19:04 --> Undefined variable: buktiPendaftaran
#0 D:\Xampp\htdocs\app-pmb2\app\Views\v_pendaftaran\nde.php(35): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\Pendaftaran.php(550): view()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pendaftaran->nde()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-11-28 02:52:53 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 02:54:42 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:03:01 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:05:45 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:30:10 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:30:54 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:32:20 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:37:48 --> Too few arguments to function App\Controllers\DataMentor::ajaxDataMentor(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->ajaxDataMentor()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:39:45 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:40:50 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:46:05 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:47:12 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:48:00 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:55:19 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:55:41 --> Undefined variable: nama_bidang
#0 D:\Xampp\htdocs\app-pmb2\app\Views\v_dataMentor\index.php(11): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataPendaftaran.php(62): view()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataPendaftaran->index()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-11-28 03:56:30 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 03:57:35 --> Undefined variable: idBidang
#0 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(49): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#4 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-11-28 03:58:08 --> Too few arguments to function App\Controllers\DataMentor::index(), 0 passed in D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php on line 918 and exactly 1 expected
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#3 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#5 {main}
CRITICAL - 2022-11-28 04:02:23 --> Undefined index: nama_mentor
#0 D:\Xampp\htdocs\app-pmb2\app\Views\layouts_admin\sidebar.php(131): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(525): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Views\layouts_admin\template_admin.php(38): CodeIgniter\View\View->include()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(257): CodeIgniter\View\View->render()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#7 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataKategori.php(54): view()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataKategori->index()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#11 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-11-28 04:05:56 --> Undefined index: nama_mentor
#0 D:\Xampp\htdocs\app-pmb2\app\Views\layouts_admin\sidebar.php(122): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(525): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Views\layouts_admin\template_admin.php(38): CodeIgniter\View\View->include()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(257): CodeIgniter\View\View->render()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#7 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataKategori.php(54): view()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataKategori->index()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#11 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#12 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-11-28 04:06:38 --> include(add.php): failed to open stream: No such file or directory
#0 D:\Xampp\htdocs\app-pmb2\app\Views\v_dataMentor\index.php(25): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\app\Views\v_dataMentor\index.php(25): include()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#4 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(55): view()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->index()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#8 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#10 {main}
CRITICAL - 2022-11-28 04:16:06 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:16:12 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:17:24 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:17:24 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:17:25 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:17:46 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:18:25 --> You must use the "set" method to update an entry.
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseBuilder->validateInsert()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-11-28 04:18:28 --> You must use the "set" method to update an entry.
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseBuilder->validateInsert()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-11-28 04:19:57 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:20:01 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:21:24 --> Unknown column 'updated_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(137): CodeIgniter\Model->update()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->update()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#9 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#11 {main}
CRITICAL - 2022-11-28 04:21:40 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:43 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:43 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:43 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:43 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:44 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:44 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:44 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:21:44 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:24:39 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:24:42 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:24:42 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:26:08 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:26:08 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(90): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:58 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:58 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:59 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:59 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:59 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:29:59 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:00 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:00 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:00 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:00 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:00 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:01 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:01 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:01 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:01 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:01 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:02 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:02 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:02 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:02 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:23 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:23 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:23 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:23 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:24 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:30:44 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:35:12 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:35:18 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:35:21 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 04:35:21 --> Unknown column 'created_at' in 'field list'
#0 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(331): mysqli->query()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2249): CodeIgniter\Database\BaseConnection->query()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(755): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Model.php(574): CodeIgniter\Model->insert()
#6 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataMentor.php(89): CodeIgniter\Model->save()
#7 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataMentor->add()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#9 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#10 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#11 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#12 {main}
CRITICAL - 2022-11-28 05:10:22 --> Trying to access array offset on value of type null
#0 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataPendaftaran.php(75): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataPendaftaran->view()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#4 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-11-28 05:10:23 --> Trying to access array offset on value of type null
#0 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataPendaftaran.php(75): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataPendaftaran->view()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#4 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-11-28 05:12:32 --> Trying to access array offset on value of type null
#0 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataPendaftaran.php(75): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataPendaftaran->view()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#4 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-11-28 05:17:18 --> Trying to access array offset on value of type null
#0 D:\Xampp\htdocs\app-pmb2\app\Controllers\DataPendaftaran.php(75): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\DataPendaftaran->view()
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#3 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#4 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-11-28 20:08:23 --> Undefined variable: nama_mentor
#0 D:\Xampp\htdocs\app-pmb2\app\Views\v_pendaftaran\dashboard.php(175): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\Pendaftaran.php(505): view()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pendaftaran->dashboard()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
CRITICAL - 2022-11-28 20:09:36 --> Undefined variable: nama_mentor
#0 D:\Xampp\htdocs\app-pmb2\app\Views\v_pendaftaran\dashboard.php(175): CodeIgniter\Debug\Exceptions->errorHandler()
#1 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\View\View.php(246): include('D:\\Xampp\\htdocs...')
#2 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render()
#3 D:\Xampp\htdocs\app-pmb2\app\Controllers\Pendaftaran.php(508): view()
#4 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Pendaftaran->dashboard()
#5 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController()
#6 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest()
#7 D:\Xampp\htdocs\app-pmb2\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 D:\Xampp\htdocs\app-pmb2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('D:\\Xampp\\htdocs...')
#9 {main}
